Source/ReadEosTable.cpp

```diff
@@ -0,0 +1,325 @@
+#include <iostream>
+
+#include <AMReX_GpuAllocators.H>
+
+#define H5_USE_16_API 1
+#include "hdf5.h"
+
+#include "EosTable.H"
+
+// mini NoMPI
+#define HAVE_CAPABILITY_MPI //FIXME: This should be defined only when USE_MPI = TRUE
```

Comment:
This causes the code to break whenever MPI is not enabled. This should be addressed.

Reply:
Done!